.eslintrc.js

```diff
@@ -13,11 +13,17 @@ module.exports = {
 	ignorePatterns: [
 		'**/dist/*',
 		'**/coverage/**',
-		'**/node_modules/**'
+		'**/node_modules/**',
+
+		// ESLint does not understand `import ... with { ... }`.
+		// See: https://github.com/eslint/eslint/discussions/15305.
```

Comment:
TODO: To resolve.

Reply:
It turns out that you need to install additional parsers that may conflict with TypeScript. Let's leave it as it is.